src/ui/PageHeading/PageHeading.tsx

```diff
@@ -0,0 +1,77 @@
+import { forwardRef } from "react";
+import styled from "styled-components";
+import shapesPink1Url from "./shapes-pink-1.svg";
+import shapesPink2Url from "./shapes-pink-2.svg";
+import shapesBlue1Url from "./shapes-blue-1.svg";
+import { H1, Subheading1, P } from "../../ui/Typography";
+
+type BackgroundType = "pink-1" | "pink-2" | "blue-1";
+
+const Wrap = styled.div<{
+  $backgroundType: BackgroundType;
+}>`
+  padding: 80px 40% 80px 11%;
```

Comment:
if we want to keep right side of the panel empty, best way is to have a row in flexbox layout divided into 2 columns, with the right column stretch in portion by specifying `flex-grow` and `flex-shrink`

if we don't want that, we can let text flow to the right when device screen is smaller